benchmarks/bench_evolve.py

```diff
@@ -33,16 +33,12 @@ def benchmark_write_operator_card_from_file(tmp_path, test_files, test_configs):
     # Check if it contains all the information for eko
     assert np.allclose(myopcard["rotations"]["xgrid"], x_grid)
     # Checking that inputpids and targetpids have been correctly removed
-    with pytest.raises(KeyError):
-        myopcard["rotations"]["inputpids"]
-        myopcard["rotations"]["targetpids"]
-        myopcard["inputpids"]
-        myopcard["targetpids"]
-        myopcard["rotations"]["inputgrid"]
-        myopcard["rotations"]["targetgrid"]
-        myopcard["inputgrid"]
-        myopcard["targetgrid"]
-
+    to_check = ["inputpids", "targetpids", "inputgrid", "targetgrid"]
+    for elem in to_check:
+        with pytest.raises(KeyError):
+            myopcard["elem"]
```

Comment:
```suggestion
            myopcard[elem]
```

:upside_down_face:

Reply:
`pytest.raises` also allows you to specify what do you expect to be contained in the error message. Add that the missing key should be actually `elem` or something like (if you were consistently quoting, it would have worked the same, but at least you should have made two consistent mistakes xD)